yarn.lock

```diff
@@ -1739,11 +1747,12 @@
   resolved "https://registry.yarnpkg.com/@types/prop-types/-/prop-types-15.7.4.tgz#fcf7205c25dff795ee79af1e30da2c9790808f11"
   integrity sha512-rZ5drC/jWjrArrS8BR6SIr4cWpW09RNTYt9AMZo3Jwwif+iacXAqgVjm0B0Bv/S1jhDXKHqRVNCbACkJ89RAnQ==
 
-"@types/react-native@0.67.6":
-  version "0.67.6"
-  resolved "https://registry.yarnpkg.com/@types/react-native/-/react-native-0.67.6.tgz#9a7de5feba6065aec9f44f9a1e8f6e55ee5d015c"
-  integrity sha512-NM6atxrefIXMLE/PyQ1bIQjQ/lWLdls3uVxItzKvNUUVZlGqgn/uGN4MarM9quSf90uSqJYPIAeAgTtBTUjhgg==
+"@types/react-native@0.72.2":
```

Comment:
Maybe we can remove this dependency completely now that `react-native` ships with types?

Reply:
In order to do that, we'd need to declare `react-native` as a devDependency instead. It's a more future-proof alternative, I believe, as I'm not really sure how long the `@types/react-native` lib will be actively maintained now that we have official support for TS in the original repo.

Reply:
> It's a more future-proof alternative, I believe, as I'm not really sure how long the @types/react-native lib will be actively maintained now that we have official support for TS in the original repo.

IIRC, that dependency will no longer be updated from 0.73 onwards. I agree we can keep as-is for this PR, though.